pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -146,9 +169,12 @@ func NewPolicyApplier(jwtPolicies []*model.Config, policy *authn_alpha_api.Polic
 	})
 
 	return &v1beta1PolicyApplier{
-		jwtPolicies:       jwtPolicies,
-		processedJwtRules: processedJwtRules,
-		alphaApplier:      alpha_applier.NewPolicyApplier(policy),
+		jwtPolicies:            jwtPolicies,
+		peerPolices:            peerPolicies,
```

Comment:
do you need peerpolicies? seems to me you only need to store `consolidatePeerPolicy` in the applier.